examples/aws/backend.ipynb

```diff
@@ -0,0 +1,147 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Packages your Python function, Python file or Jupyter notebook as a Docker image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Environment Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import time\n",
+    "from kubeflow import fairing\n",
+    "from kubeflow.fairing import TrainJob\n",
+    "from kubeflow.fairing.backends import KubeflowAWSBackend"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "AWS_ACCOUNT_ID='348134392524'\n",
```

Comment:
Set to the variable reminder name such as `Your_AWS_ACCOUNT_ID`?

Reply:
I may forget to clean up this one. Thanks! Delete aws id and get it from library